app/views/content/explore-teaching-advisers/_ending.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row inset">
```

Comment:
This section doesn't appear on the front end @plisovin - it should?

https://review-get-into-teaching-app-3472.london.cloudapps.digital/explore-teaching-advisers

Instead the teacher-training-advisers ending appears instead (sections titled `If you are a veteran` and `Not a final year student or graduate`? This needs removing and the explore-teaching-advisers ending needs adding.

Reply:
> Instead the teacher-training-advisers ending appears instead (sections titled `If you are a veteran` and `Not a final year student or graduate`? This needs removing and the explore-teaching-advisers ending needs adding.

Sorry, was wrong partial used in ETA, fixed now